Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -513,6 +513,12 @@ define([
             this._model._clippingPlanes = (tilesetClippingPlanes.enabled && this._tile._isClipped) ? tilesetClippingPlanes : undefined;
         }
 
+        // If the model references a destroyed ClippingPlaneCollection due to the tileset's collection being replaced with a
+        // ClippingPlaneCollection that gives this tile the same clipping status, update the model to use the new ClippingPlaneCollection.
+        if (defined(tilesetClippingPlanes) && defined(this._model._clippingPlanes) && this._model._clippingPlanes.isDestroyed()) {
```

Comment:
Alternatively could the check be `tilesetClippingPlanes !== this._model._clippingPlanes`?

Once https://github.com/AnalyticalGraphicsInc/cesium/issues/6599 is fixed we won't be destroying the collection anymore.